Word/WordDocumentGenerator.cs

```diff
@@ -233,35 +276,51 @@ private static void ReplaceImagePlaceholders(string inputFilePath, string output
                     // If drawing / image name is present in imagePlaceholders dictionary, then replace image
                     if (docProperty != null && imagePlaceholders.ContainsKey(docProperty.Name))
                     {
-                        List<Blip> drawingBlips = drawing.Descendants<Blip>().ToList();
+                        string imagePath = imagePlaceholders[docProperty.Name];
 
-                        foreach (Blip blip in drawingBlips)
-                        {
-                            OpenXmlPart imagePart = wordDocument.MainDocumentPart.GetPartById(blip.Embed);
-
-                            using (var writer = new BinaryWriter(imagePart.GetStream()))
-                            {
-                                string imagePath = imagePlaceholders[docProperty.Name];
-
-                                /*
-                                 * WebClient has been deprecated and we need to use HTTPClient.
-                                 * This involves the methods to be asynchronous.
-                                 */
-                                using (WebClient webClient = new WebClient())
-                                {
-                                    writer.Write(webClient.DownloadData(imagePath));
-                                }
-                            }
-                        }
+                        // Replace the image with the new image
+                        ReplaceImage(mainDocumentPart, drawing, imagePath);
                     }
                 }
             }
 
-            // Overwrite the output file
-            FileStream fileStream = new FileStream(outputFilePath, FileMode.OpenOrCreate, FileAccess.ReadWrite);
-            memoryStream.WriteTo(fileStream);
-            fileStream.Close();
+            // Save the updated document bytes to the output file path
```

Comment:
Are these the changes that were suggested by the editorconfig rules?

Reply:
no, i verified the code with the main branch, made the necessary changes to my repository. My code was not updated with the latest commit.